checkRFC4566.js

```diff
@@ -21,7 +21,7 @@ const blankLines = /\r\n?\r|\n\r?\n/;
 const linePattern = /^([a-z]=\S.*|s= )$/;
 const letterCheck = /^[vosiuepcbzkatrm].*$/;
 // const spaceCheck = /\s\s/;
-const mustHaves = [ 'v', 'o', 's', 't' ];
+const mustHaves = [ 'v', 'o', 's', 't', 'm' ];
```

Comment:
Corrected this - my misreading of the spec since in the section it states that items with * are optional and the subsection describing m does not have the *. However it's clear that it is optional on closer reading. I found that we can set the parameter --noMedia in the original code and the tests will look for the optional m=video.  Tested that and it works.  Committing that change.

Reply:
@garethsb Marking this one as resolved this followed your correction